File: 백준/Bronze/1110. 더하기 사이클/더하기 사이클.js
const fs = require('fs');
const filePath = process.platform === 'linux' ? 'dev/stdin' : 'b/example.txt';
let input = +fs.readFileSync(filePath).toString().trim();
let sum = 0;
let num = input;
let count = 0;

while (true) {
  sum = Math.floor(num / 10) + (num % 10);
  num = Number(String(num % 10) + (sum % 10));
  count++;

  if (num === input) break;
}

console.log(count);

File: 백준/Bronze/1110. 더하기 사이클/더하기 사이클.py
input_num = int(input())


num = input_num
sum = 0
count = 0

while True:
    sum = (num // 10) + (num % 10)
    num = int(str(num % 10) + str(sum % 10))
    count += 1

    if num == input_num:
        break

print(count)
